torchsupport/modules/rezero.py

```diff
@@ -0,0 +1,18 @@
+import torch
+import torch.nn as nn
+import torch.nn.functional as func
+
+class ReZero(nn.Module):
+  r'''Implemets ReZero normalization proposed by Bachlechner et al. 
+    (https://arxiv.org/pdf/2003.04887.pdf) 
+  Args: 
+    function (callable): function you want to use (for example conv2D)'''
+  def __init__(self, function):
+    super().__init__()
+    self.function = function
+    self.alpha = nn.Parameter(torch.tensor(
+      0, dtype=torch.float, requires_grad=True
```

Comment:
Sounds cool! From the paper I'm not sure if their alpha is a single scalar per layer or per channel (as popular with normalizations)

Reply:
It seems its a single scalar, but nothing stops us from adding an optional argument with a size for the alpha which defaults to a scalar. Sounds like a natural extension if you ask me.